### ISSUE TITLE
Fetal scalp mass

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7370]
**Preferred term label:**
Fetal scalp mass

**Synonyms**

**Definition (free text, please give PubMed ID)**
An abnormal growth (lump) of tissue that projects from the fetal scalp.
PMID: 24767653
**Parent term (use hpo.jax.org/app)**
need to add fetal head morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
